### ISSUE TITLE
Integration settings not viewable in production environment

### ISSUE THREAD

**Issue by mtnorthrop** [verbb/formie#995]
### Describe the bug

I am not able to see integration instructions and settings in an environment where `allowAdminChanges` has been set to `false`.

For example, I'm trying to view the Constant Contact connection instructions and, most importantly, the redirect URL that Formie generates, in the production environment, but instead I just see the `Integration settings can only be editable on an environment with allowAdminChanges enabled` warning.

I would like my client to be able to see the connection instructions and the redirect URL so they can configure their Constant Contact application correctly. (I can't configure this in my dev environment because I do not have access to the client's Constant Contact account.)

I understand why the configuration should not be changed when `allowAdminChanges` is set to `false`, but I'm reporting this as a bug because the connection instructions and redirect URL _used_ to appear in the production environment but now they don't. I just updated from 1.5.16 to 1.6.4, and this information was available, even when allowAdminChanges was disabled, in version 1.5.16.

This looks like a regression to me, because, based on https://github.com/verbb/formie/issues/159#issuecomment-707110527, it appears that Formie is meant to display these details (read-only) in production environments.

### Steps to reproduce

(See above)

### Form settings

N/A

### Craft CMS version

3.7.48

### Plugin version

1.6.4

**Comment by engram-design** [comment]
Ah yes, good point. For any OAuth-based integrations, we shouldn't show this message. The tricky part is that people are editing these values and clicking "connect" thinking that their updated values are being saved when they aren't. The integration connection details are a sticky point where they should and shouldn't be stored in project config.

But this is probably a bit too heavy-handed hiding some of these things. I would go through every integration and add a `readonly` attribute to the inputs for these fields, but this doesn't work for selects, autocomplete inputs or lightswitches. The only resolution is to hide the fields, or at least display a warning.

Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by mtnorthrop** [comment]
@engram-design , that's great, thank you very much! I appreciate the problem with displaying fields in a production environment; I wonder if it would be possible to show the data (like redirect URL) as plain text, instead of in a field? But maybe this would just be extra work that isn't worth the effort...

**Comment by engram-design** [comment]
At the moment, we just show it (the Redirect URI field) in a read-only field, which I think will suffice, but I'll consider doing the same for other settings.

**Comment by engram-design** [comment]
Should be fixed in [1.6.5](https://github.com/verbb/formie/releases/tag/1.6.5)
